### ISSUE TITLE
2021-6-1

### ISSUE THREAD

**Issue by ssfc** [ssfc/wangdao#561]
571为什么大型机器学习模型必须缩小 ？ https://mp.weixin.qq.com/s/upC6vRpJqqnckaOFUwfO0w (2021-5-19)
572Skip-gram是用中心词预测周围词, cbow使用周围词预测中心词; 简单说，skip-gram 出来的准确率比cbow 高; Skip-gram: works well with a small amount of the training data, represents well even rare words or phrases. CBOW: several times faster to train than the skip-gram, slightly better accuracy for the frequent word; (2021-5-19)
573What is python nltk library? The Natural Language Toolkit, or more commonly NLTK, is a suite of libraries and programs for symbolic and statistical natural language processing (NLP) for English written in the Python programming language. It was developed by Steven Bird and Edward Loper in the Department of Computer and Information Science at the University of Pennsylvania; (2021-5-19)
574How to install nltk library in python? 
575微软机器学习最新进展 https://mp.weixin.qq.com/s/yJ1qiHGTZTgHwuvaaTR7ew (2021-5-20)
576【综述】一文概览动态神经网络 https://mp.weixin.qq.com/s/7BEMBZMejYR1UV5qcENoLg (2021-5-20)
577思维的极限：科学的进步将取决于人类还是AI ？ https://mp.weixin.qq.com/s/G1Wtu7sWeyxXRWFJLkjcFA (2021-5-20)
578Try to output data in a text file; add button Export Sentence in analyzer; done; give update abundance function to button Export Sentence; done; separate file name; done; order by ID; done; (2021-5-20)
579What if torch.matmul contain a vector? If both tensors are 1-dimensional, the dot product (scalar) is returned. If both arguments are 2-dimensional, the matrix-matrix product is returned. If the first argument is 1-dimensional and the second argument is 2-dimensional, a 1 is prepended to its dimension for the purpose of the matrix multiply. After the matrix multiply, the prepended dimension is removed. If the first argument is 2-dimensional and the second argument is 1-dimensional, the matrix-vector product is returned. (2021-5-20)
